src/index.ts

```diff
@@ -11,5 +11,6 @@ export function configure(config: FrameworkConfiguration) {
     .feature(PLATFORM.moduleName('oerelia/systemfields/index'))
     .feature(PLATFORM.moduleName('oerelia/telefoon-select/index'))
     .feature(PLATFORM.moduleName('oerelia/adres-crab/index'))
-    .feature(PLATFORM.moduleName('oerelia/spinner/index'));
+    .feature(PLATFORM.moduleName('oerelia/spinner/index'))
+    .feature(PLATFORM.moduleName('oerelia/multi-select/index'));
```

Comment:
Value-converters index file aanmaken en ook featuren volgens mij om het te kunnen gebruiken in de html templates

Reply:
had ik al eens geprobeerd maar doordat dit een value converter is kon die dat niet goed resolven dacht ik. Of is dat in andere projecten wel gelukt? Zal nog eens kijken

Reply:
dan krijg ik deze `Possible Unhandled Promise Rejection: Error: No ValueConverter named "dateFormat" was found!` misschien dat ik iets over het hoofd zie hier, zal even door blijven zoeken

Reply:
Er is er nog geen enkele extern gebruikt nee, zou idd kunnen dat dat voor issues zorgt. Als het zo werkt is het in orde, was maar een bedenking :)

Reply:
fixed, ik ontbrak nog een `config.globalResources` definitie in die index van de value-converters, zie https://github.com/OnroerendErfgoed/oerelia/pull/36/commits/e4d8dfbad09df7710bd899d7a0ee00b1e0d9563f